docs/writing-tests/unit-testing.md

```diff
@@ -0,0 +1,137 @@
+
+<!-- prettier-ignore-start -->
+
+<CodeSnippets
+  paths={[
+    'react/storybook-testing-addon-optional-config-scripts.json.mdx',
+  ]}
+/>
+
+<!-- prettier-ignore-end -->
+
+### Override story properties
+
+By default, the `setProjectAnnotations` function injects into your existing tests any global configuration you've defined in your Storybook instance (i.e., parameters, decorators in the `preview.js|ts` file). Nevertheless, this may cause unforeseen side effects for tests that are not intended to use these global configurations. To avoid this, you can override the global configurations by extending either the `composeStory` or `composeStories` functions to provide test-specific configurations. For example:
+
+<!-- prettier-ignore-start -->
+
+<CodeSnippets
+  paths={[
+    'react/override-compose-story-test.compose-story.js.mdx',
+    'react/override-compose-story-test.compose-story.ts.mdx',
+    'react/override-compose-story-test.compose-stories.js.mdx',
+    'react/override-compose-story-test.compose-stories.ts.mdx',
+  ]}
+/>
+
+<!-- prettier-ignore-end -->
+
+## Run tests on a single story
+
+You can use the `composeStory` function from the appropriate framework or supported addon to allow your tests to run on a single story. However, if you're relying on this method, we recommend that you supply the story metadata (e.g., the [default export](../writing-stories/introduction.md#default-export)) to the `composeStory` function. This ensures that your tests can accurately determine the correct information about the story. For example:
+
+<!-- prettier-ignore-start -->
+
+<CodeSnippets
+  paths={[
+    'react/single-story-test.js.mdx',
+    'react/single-story-test.ts.mdx',
+  ]}
+/>
+
+<!-- prettier-ignore-end -->
+
+## Combine stories into a single test
+
+If you intend to test multiple stories in a single test, use the `composeStories` function from the appropriate framework or supported addon. The function will process every component story you've specified, including any [`args`](../writing-stories/args.md) or [`decorators`](../writing-stories/decorators.md) you've defined. For example:
+
+<!-- prettier-ignore-start -->
+
+<CodeSnippets
+  paths={[
+    'react/multiple-stories-test.js.mdx',
+    'react/multiple-stories-test.ts.mdx',
+  ]}
+/>
+
+<!-- prettier-ignore-end -->
+
+## Troubleshooting
+
+### Run tests in other frameworks
+
+Storybook provides community-led addons for other frameworks like [Vue 2](https://storybook.js.org/addons/@storybook/testing-vue), [Vue 3](https://storybook.js.org/addons/@storybook/testing-vue3) and [Angular](https://storybook.js.org/addons/@storybook/testing-angular). However, these addons still lack support for the latest stable Storybook release. If you're interested in helping out, we recommend reaching out to the maintainers using the default communication channels (GitHub and [Discord server](https://discord.com/channels/486522875931656193/839297503446695956)).
```

Comment:
testing-vue3 now supports Storybook 7! but the state of the others remain the same